README.md

```diff
@@ -4,18 +4,25 @@ This repository contains the default Excessive Motion UI frontend and a minimal
 ### Project Structure
 - TODO: this (Webpack, TypeScript, ...)
 
-### Build Requirements
-- Node.js
-
-## Development
 The **Frontend** and **CEF App** projects are set up to work from the root reporitory directory, so no directory changes are required to build/run both.
 
-### Frontend [Webpack/TypeScript]
-Before starting, run ```npm update``` to download all of the development dependencies.
-- Run ```npm run start``` to start a live development server
-- Run ```npm run build``` to build the UI in development mode
-- Run ```npm run dist``` to build the UI in production mode
+## Frontend [Webpack/TypeScript]
+### Build requirements
+- Node.js
+  - Install `nvm` by following the instructions here: https://github.com/nvm-sh/nvm#installing-and-updating
+- pnpm
+  - Install using ```corepack```:
+```sh
+corepack enable
+corepack prepare pnpm@latest
+```
+
+### Development
+Before starting, run ```pnpm install``` to download all of the development dependencies.
+- Run ```pnpm start``` to start a live development server at localhost:8080
```

Comment:
Small detail, but the live server is not always started on :8080 (if :8080 is already in use, the server will be started on the next available port)